Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM python:3.10-slim
+
+RUN pip3 install --no-cache-dir --upgrade \
+    pip \
+    virtualenv
+
+RUN apt-get update && apt-get install -y \
+    build-essential \
+    software-properties-common \
+    git
+
+    ENV VIRTUAL_ENV=/home/venv
```

Comment:
Hey! Maybe I'm missing something, but is there a specific reason to use virtualenvs?

Reply:
No reason, just copy-pasted and modified what streamlit recommended here https://docs.streamlit.io/deploy/tutorials/kubernetes#create-a-dockerfile . I am pretty sure the `appuser` is also kind of irrelevant

Reply:
Gotcha, yeah, the user could make sense if you want to avoid `root`. In that case, please, delete the virtualenv related configuration.

Reply:
Done!

Reply:
@daavoo it's not done, the virtual env stills there, please, review it